README.md

```diff
@@ -35,6 +36,36 @@ end
 
 Then run `bundle exec rake test`
 
+### Build failure criteria
+
+By default, `ra11y` will cause a build to fail if any errors or warnings are generated. For some users, this might be too strict. `pa11y` allows you to customize the failure criteria with a `Hash`:
```

Comment:
I usually don't put gem names in code blocks. Would you put `Microsoft Word` in a code block?